scripts/tcga_fpkm_dendrogram/README.md

```diff
@@ -0,0 +1,53 @@
+# TCGA_FPKM_dendrogram
```

Comment:
In general, we want this script to be a generic dendogram + PCA clustering script, not necessarily focused on the TCGA data set. So it's best not to mention TCGA in the documentation or in the code.